src/app/commissioner_app.cpp

```diff
@@ -135,11 +135,14 @@ const BorderAgent *CommissionerApp::GetBorderAgent(const std::string &aNetworkNa
     return nullptr;
 }
 
-Error CommissionerApp::Start(const std::string &aBorderAgentAddr, uint16_t aBorderAgentPort)
+Error CommissionerApp::Start(std::string &      aActiveCommissionerId,
+                             const std::string &aBorderAgentAddr,
+                             uint16_t           aBorderAgentPort)
 {
     Error error = Error::kNone;
 
-    SuccessOrExit(error = mCommissioner->Petition(aBorderAgentAddr, aBorderAgentPort));
+    // We need to report the active commissioner ID even when pulling network data failed.
```

Comment:
Is it possible to report the Commissioner ID of this commissioner (i.e. when there was not an existing active commissioner)? If so, that might be confusing for the user.

Reply:
No, the changes at [here](https://github.com/openthread/ot-commissioner/pull/36/files#diff-664c9f59b79e0e70d3f1e8d022c9b49bR1126) guarantees that we only set the Commissioner ID when we are rejected (there are existing active commissioner).

Reply:
Okay, the comment seems confusing to me. Can you clarify what is intended by the comment?